components/calendar/src/iso.rs

```diff
@@ -306,7 +306,7 @@ impl DateTime<Iso> {
                 + i32::from(self.time.hour.number()) * minutes_a_hour
                 + i32::from(self.time.minute.number())
         } else {
-            0
+            -1
```

Comment:
Use -1 here so we will return `None` for local timestamp before unix epoch while calculating metazone id.

Reply:
I think it is unreachable though? It is the error case in

```rust
        if let Ok(unix_epoch) = DateTime::new_iso_datetime(1970, 1, 1, 0, 0, 0) {
```

which is constant.

Reply:
oh yeah, I misread the code.